iina/LogWindowController.swift

```diff
@@ -39,6 +39,9 @@ class LogWindowController: NSWindowController, NSMenuDelegate {
     }
     levelPopUpButton.selectItem(withTag: Logger.Level.preferred.rawValue)
     subsystemPopUpButton.menu!.delegate = self
+
+    Timer.scheduledTimer(timeInterval: 0.1, target: self, selector: #selector(syncLogs), userInfo: nil, repeats: true)
```

Comment:
After thinking on this some more, I think a A non-repeating timer should be used instead of a repeating timer. Using a non-repeating timer allows IINA to go idle when logs are not being emitted without the need for extra logic. Something like the following…

When a message is logged:
Add the log to the array.
Check if a non-repeating timer has already been scheduled, if not schedule one.
This would have to be done under a lock, but locking needs to be added anyway to coordinate access to the array as discussed below.

This way IINA goes idle when playback is paused even if the log viewer is open.